ffi-build.sh

```diff
@@ -32,7 +32,7 @@ case "$target" in
         native_static_libs="${expected_native_static_libs}"
         ;;
     "x86_64-unknown-linux-gnu")
-        expected_native_static_libs=" -ldl -lrt -lpthread -lgcc_s -lc -lm -lrt -lpthread -lutil -ldl -lutil"
+        expected_native_static_libs=" -ldl -lrt -lpthread -lgcc_s -lc -lm -lutil"
         native_static_libs=" -ldl -lrt -lpthread -lc -lm -lrt -lpthread -lutil -ldl -lutil"
```

Comment:
Thse two lines should be exactly the same except that `native_static_libs` removes `-lgcc_s` in the output, I think?

Reply:
This change was purely cosmetic, @r1viollet suggested it was no longer relevant to repeat the libraries since we remove duplicated elements later on ;)
I guess both possibilities can be argued for:
* keep the output of `--print=native-static-libs` as this
* keep only unique elements since order and repetitions have no impact

Reply:
Ah yes ! This was my fault. My thought-process is that I want to check what is being linked. The repetition is somewhat of a detail that I don't need to see. 
I understand the value of just copy pasting the result. 
Feel free to choose either.